src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/BeneficialOwnersStatementType.java

```diff
@@ -3,29 +3,36 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 
 public enum BeneficialOwnersStatementType {
+  ALL_IDENTIFIED_ALL_DETAILS("all_identified_all_details"),
+  SOME_IDENTIFIED_ALL_DETAILS("some_identified_all_details"),
+  NONE_IDENTIFIED("none_identified");
 
-    ALL_IDENTIFIED_ALL_DETAILS("all_identified_all_details"),
-    SOME_IDENTIFIED_ALL_DETAILS("some_identified_all_details"),
-    NONE_IDENTIFIED("none_identified");
+  private final String beneficialOwnersStatement;
 
-   private final String beneficialOwnersStatement;
+  BeneficialOwnersStatementType(String beneficialOwnersStatement) {
+    this.beneficialOwnersStatement = beneficialOwnersStatement;
+  }
 
-   BeneficialOwnersStatementType(String beneficialOwnersStatement) {
-        this.beneficialOwnersStatement = beneficialOwnersStatement;
+  /**
+   * Json creator allows non-matching enum values to be interpreted by the method which determines
+   * them as null
+   *
+   * @param beneficialOwnersStatement
+   * @return
+   */
+  @JsonCreator
+  public static BeneficialOwnersStatementType findByBeneficialOwnersStatementTypeString(
+      String beneficialOwnersStatement) {
+    for (BeneficialOwnersStatementType type : values()) {
+      if (type.beneficialOwnersStatement.equalsIgnoreCase(beneficialOwnersStatement)) {
+        return type;
+      }
     }
+    return null;
+  }
 
-    /**
-     * Json creator allows non-matching enum values to be interpreted by the method which determines them as null
-     * @param beneficialOwnersStatement
-     * @return
-     */
-   @JsonCreator
-   public static BeneficialOwnersStatementType findByBeneficialOwnersStatementTypeString(String beneficialOwnersStatement) {
-       for (BeneficialOwnersStatementType type: values()) {
-           if(type.beneficialOwnersStatement.equalsIgnoreCase(beneficialOwnersStatement)) {
-               return type;
-           }
-       }
-       return null;
-   }
+  /* getValue() returns the selected value (for e.g. in Update Submission) from this enum list */
```

Comment:
Please update the format of this javadoc, especially since we're returning something

```suggestion
  /**
   * getValue() returns the selected value (for e.g. in Update Submission) from this enum list 
   * @return String
   */
```